packages/salmon-price-functions/src/price_manager.ts

```diff
@@ -20,9 +20,6 @@ export class PriceManager {
     private readonly config: PriceSourceConfig,
```

Comment:
```suggestion
    readonly config: PriceSourceConfig,
```
Shall we have it public? I think allow checking config for instantiated PriceManager is useful.

Reply:
Yes ok